src/index.css

```diff
@@ -0,0 +1,48 @@
+/* Tailwind imports */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* Font import */
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap");
+
+/* General styles */
+html,
+body,
+ul,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
You don't need this as tailwind does this automatically in preflight: https://tailwindcss.com/docs/preflight